pkg/test/resourcefixture/testdata/basic/secretmanager/v1beta1/secretmanagersecretversion/update.yaml

```diff
@@ -20,10 +20,5 @@ metadata:
   name: secretmanagersecretversion-${uniqueId}
 spec:
   enabled: false
-  secretData:
```

Comment:
Well I think it's a good test, but I would expect if I removed the secretData that KCC would remove the secret data.  So we might not want to open this can of worms just yet